docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -73,7 +73,7 @@ to the above command (i.e. the path to the private key).
       $ kubectl get pods
 
     will get a list of the pods in the Kubernetes cluster associated
-    with the context named ``k8s-bdb-test-cluster-0``.
+    with the context named ``k8s-bdb-test-node-0``.
 
 Step 2: Connect to Your Cluster's Web UI (Optional)
```

Comment:
Should this say "Your Cluster's Web UI" or "Your Node's Web UI"? Technically, it _is_ a Kubernetes cluster (but the cluster is just for one node)...

Reply:
Agreed.